godot/convert.go

```diff
@@ -71,6 +71,10 @@ var goToGodotConversionMap = map[string]goToGodotConverter{
 		arg := goObject.(*Quat)
 		return unsafe.Pointer(arg.quat)
 	},
+	"*Real": func(goObject interface{}) unsafe.Pointer {
+		arg := goObject.(*Real)
```

Comment:
We don't have a Go structure called `Real`.

Reply:
I'll remove it on the next commit. I mistakenly understood it to work the opposite direction.